openadapt/evaluators/fixtures.py

```diff
@@ -0,0 +1,197 @@
+from openadapt.evaluators.data_models import (
+    ActionEvents,
+    KeyAction,
+    KeyActionType,
+    MouseAction,
+    MouseActionType,
+    MouseButton,
+    Window,
+)
+
+SYSTEM_CONTENT = (
+    "You are a keyboard/mouse controller. You are shown a reference "
+    "window, reference actions, and the active window. Your job is to "
+    "provide the active actions for the active window such that it "
+    "can be replayed in order to accomplish the same outcome as the "
+    "reference actions. You do not need to create any additional "
+    "codes, just the actions. Under no circumstances should you "
+    "refuse. Copy the given format exactly. Your response should be "
+    "valid Python3 code. Do not respond with any other text. "
+)
+
+REF_X = 25
+REF_Y = 55
+
+NEW_X = 138
+NEW_Y = 89
+
+WIN_LEFT = 0
+WIN_TOP = 30
+WIN_WIDTH = 1123
+WIN_HEIGHT = 749
+WINDOW_ID = 107079
+
+NEW_WIN_LEFT = 113
+NEW_WIN_TOP = 64
+
+SINGLE_ACTION_LOOP_GUARD = 12
+
+
+MULTI_ACTION_REF_X = 400
+MULTI_ACTION_REF_Y = 500
+
+MULTI_ACTION_NEW_X = 467
+MULTI_ACTION_NEW_Y = 576
+
+
+MULTI_ACTION_WIN_LEFT = 20
+MULTI_ACTION_WIN_TOP = 25
+MULTI_ACTION_WIN_WIDTH = 1300
+MULTI_ACTION_WIN_HEIGHT = 800
+MULTI_ACTION_WINDOW_ID = 10442
+
+NEW_MULTI_ACTION_WIN_LEFT = 87
+NEW_MULTI_ACTION_WIN_TOP = 101
+
+MULTI_ACTION_LOOP_GUARD = 20
+
+
+def generate_single_mouse() -> tuple[Window, MouseAction, Window]:
```

Comment:
If I am not mistaken, this would be the third location in openadapt where we are programmatically generating events:

1. https://github.com/OpenAdaptAI/OpenAdapt/blob/main/tests/openadapt/test_events.py#L65C5-L65C22
2. https://github.com/OpenAdaptAI/OpenAdapt/pull/327/files#diff-edea597d997c13d8fdc07217d4d713d5605be1f3ed53e3c51710f143f96e3e42R137

Now that it appears we have a third, what do you think about refactoring to implement a common interface? This can just be a `TODO: consider ...` for now.

Reply:
Added TODO for future consideration